Editor/EditorCore/BezierMeshOverlay.cs

```diff
@@ -0,0 +1,201 @@
+            {
+                name = "Bezier Mesh Overlay"
+            };
+
+            if (s_StyleSheet == null)
+                s_StyleSheet = AssetDatabase.LoadAssetAtPath<StyleSheet>(k_PathToStyleSheet);
+
+            root.styleSheets.Add(s_StyleSheet);
+
+            root.Add(m_SegmentIntSlider);
+            root.Add(m_RadiusFloatSlider);
+            root.Add(m_FacesIntSlider);
+
+            return root;
+        }
+
+        /// <summary>
+        /// Initializes the slider and input fields, as well as registering to the callbacks to update their respective
+        /// parameters in the <see cref="BezierMesh"/> that they are linked to.
+        /// </summary>
+        void InitSliderAndInputFields()
+        {
+            m_SegmentIntSlider = new IntSlider("Segments per Unit", BezierMesh.k_SegmentsMin,
+                BezierMesh.k_SegmentsMax)
+            {
+                tooltip = L10n.Tr("Number of length-wise segments of the mesh per unit length")
+            };
+            m_SegmentIntSlider.slider.RegisterValueChangedCallback(evt => UpdateMeshSegments());
+
+            m_RadiusFloatSlider =
+                new FloatSlider("Radius", BezierMesh.k_RadiusMin, BezierMesh.k_RadiusMax)
+                {
+                    tooltip = L10n.Tr("The distance of the mesh from the center of the spline")
+                };
+            m_RadiusFloatSlider.slider.RegisterValueChangedCallback(evt => UpdateMeshRadius());
+
+            m_FacesIntSlider =
+                new IntSlider("Faces per Segment", BezierMesh.k_FacesMin, BezierMesh.k_FacesMax)
+                {
+                    tooltip = L10n.Tr("The number of faces around the bezier mesh at each segment")
+                };
+            m_FacesIntSlider.slider.RegisterValueChangedCallback(evt => UpdateMeshFaces());
+        }
+
+        /// <summary>
+        /// Updated the visibility of the overlay based on if at least one GameObject with a <see cref="BezierMesh"/>
+        /// is currently selected.
+        /// </summary>
+        void OnSelectionChanged()
+        {
+#if !UNITY_2022_1_OR_NEWER
+            var hasBezierMesh = false;
+            m_SelectedMeshes.Clear();
+            foreach (var selected in Selection.gameObjects)
+            {
+                if (selected.TryGetComponent(out BezierMesh mesh))
+                {
+                    m_SelectedMeshes.Add(mesh);
+                    hasBezierMesh = true;
+                }
+            }
+
+            m_Visible = hasBezierMesh;
+#endif
+
+            if (m_FacesIntSlider == null || m_RadiusFloatSlider == null ||
+                m_SegmentIntSlider == null || m_SelectedMeshes.Count == 0)
+                return;
+
+            SetParameterValues();
+        }
+
+        /// <summary>
+        /// Handles setting the parameters in the overlay to the selected Bezier Meshes' values if they are all equal,
+        /// otherwise blanks out the parameters that aren't equal.
```

Comment:
What do you mean by "blanks out"? 
```suggestion
        /// Sets the parameters in the overlay to the selected Bezier Meshes' values if they are all equal,
        /// otherwise blanks out the parameters that aren't equal.
```

Reply:
By 'blank out' I mean, instead of having a number value, there is a greyed out '-' instead.